CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+# [0.8.4] - 2021-06-20
+
+### Added
+- Add recommended labels (#320)
+- Add Storage Capacity Tracking mode (#315)
+
+### Changed
+- Add error message (#314)
+- add document to run test using minikube for lvmd daemonset (#318)
+- Remove CSI Attacher sidecar (#319)
+  - **BREAKING**: You need to recreate the CSIDriver resource because of CSI Attacher sidecar was removed from topolvm.
```

Comment:
Now I'm asking sig-storage if this approach is correct:
https://kubernetes.slack.com/archives/C09QZFCE5/p1624182034448700

Reply:
The answer from sig is here:
https://kubernetes.slack.com/archives/C09QZFCE5/p1624235246449900?thread_ts=1624182034.448700&cid=C09QZFCE5